src/libpriv/rpmostree-core.c

```diff
@@ -1328,6 +1331,23 @@ find_pkg_in_ostree (OstreeRepo     *repo,
 
       if (!same_pkg_chksum)
         return TRUE; /* Note early return */
+
+      /* We need to handle things like the nodocs flag changing; in that case we
+       * have to redownload.
+       */
+      gboolean global_docs;
+      g_variant_dict_lookup (self->spec->dict, "documentation", "b", &global_docs);
```

Comment:
Let's `g_assert` this to make it more obvious it's always set?

Edit: and in `import_one_package` too!